lib/class-wp-rest-shortcodes-controller.php

```diff
@@ -124,47 +133,16 @@ public function get_shortcode_output( $request ) {
 					wp_die( printf( '<p>An error occurred: %s (%d)</p>', $message, $error_data ) );
 				}
 				$embed_data = $embed_response->get_data();
-				$output     = $embed_data->html;
 			}
-		} else {
-			$output = do_shortcode( $shortcode );
-		}
-
-		if ( empty( $output ) ) {
-			$data['html'] = __( 'Sorry, couldn\'t render a preview', 'gutenberg' );
-			return rest_ensure_response( $data );
-		}
-
-		// Check if shortcode is returning a video. The video type will be used by the frontend to maintain 16:9 aspect ratio.
-		if ( has_shortcode( $shortcode, 'video' ) ) {
-			$type = 'video';
-		} elseif ( has_shortcode( $shortcode, 'embed' ) ) {
 			$type = $embed_data->type;
-		} else {
-			$type = 'html';
-			// Gallery and caption shortcodes need the theme style to be embedded in the shortcode preview iframe.
-			if ( has_shortcode( $shortcode, 'gallery' ) || has_shortcode( $shortcode, 'caption' ) || has_shortcode( $shortcode, 'wp_caption' ) ) {
-				$style = '<link rel="stylesheet" type="text/css" href="' . get_stylesheet_uri() . '" />';
-			}
-
-			// Playlist shortcodes need the playlist JS to be embedded in the shortcode preview iframe.
-			if ( has_shortcode( $shortcode, 'playlist' ) ) {
-				ob_start();
-				wp_footer();
-				$js = ob_get_clean();
-			}
 		}
-
 		$data = array(
-			'html'  => $output,
-			'type'  => $type,
-			'style' => $style,
-			'js'    => $js,
+			'html'                  => $output,
+			'head_scripts_styles'   => $head_scripts_styles,
```

Comment:
this is great but the schema at the bottom of the file is still reflecting `style` and `js` names

Reply:
I've updated the schema, thanks.